cardano-shell.cabal

```diff
@@ -45,30 +45,28 @@ library
     , specs
   build-depends:
       aeson
-    , base >=4.7 && <5
+    , base >=4.11 && <5
```

Comment:
Why is this changed?

Reply:
Most of the `cardano-*` projects only build with ghc-8.4 and later. Ghc-8.4 ships with `base-4.11.*`

Reply:
I tried to get rid of the contravariant package. It is included in base-4.12.
But, couldn't raise the version too high. Can be reverted, unless prelude is updated.